src/js/edu-benefits/5490/config/form.js

```diff
@@ -537,6 +537,31 @@ const formConfig = {
             educationProgram: {
               name: {
                 'ui:title': 'Name of school, university, or training facility you want to attend'
+              },
+              educationType: {
+                'ui:options': {
+                  updateSchema: (pageData, form, schema) => {
+                    const newSchema = _.cloneDeep(schema);
+                    const benefitData = _.get('benefitSelection.data.benefit', form);
+                    const relationshipData = _.get('applicantInformation.data.relationship', form);
+                    const labels = Object.keys(
+                      _.get('schoolSelection.uiSchema.educationProgram.educationType.ui:options.labels', form));
+
+                    // Remove tuition top-up
+                    const filterOut = ['tuitionTopUp'];
+                    // Correspondence not available to chapter35 children
+                    if (benefitData === 'chapter35' && relationshipData === 'child') {
+                      filterOut.push('correspondence');
+                    }
+                    // Flight training available to fry scholarships only
+                    if (benefitData !== 'chapter33') {
+                      filterOut.push('flightTraining');
+                    }
+
+                    newSchema.enum = _.without(filterOut)(labels);
```

Comment:
Yes, that's right: `labels` is the original array and is passed to `_.without()` to filter out the given items. I found this documentation useful since we are using lodash/fp, which has slightly different syntax from lodash: https://gist.github.com/jfmengels/6b973b69c491375117dc#_withoutvalues-array